ruby/vampires.rb

```diff
@@ -0,0 +1,61 @@
+puts "How many employess will we be processing today?"
+employees_processed = gets.chomp.to_i
+
+until employees_processed == 0
+	puts "What's your name?"
+	applicant_name = gets.chomp
+
+	puts "How old are you?"
+	applicant_age = gets.chomp.to_i
+
+	puts "What year were you born?"
+	applicant_year_born = gets.chomp.to_i
+		if 2017 - applicant_year_born == applicant_age

```

Comment:
I would like to see you follow the age convention in the instructions.  We want to know if you the employee is lying about their age.

Lets consider checking the current year against how old they say they are and what year they say that they were born in.  Investigate the Ruby `Time` class and see if you can come up with something creative and dynamic.